docs/logic-sandbox/calculations/2025-11-08-recovery-testing.md

```
@@ -0,0 +1,205 @@
+- Day 1: Expected 98.1%, Got 98.1% ✅
+- Day 2: Expected 83.1%, Got 83.1% ✅
+- Day 3: Expected 68.1%, Got 68.1% ✅
+- Day 4: Expected 53.1%, Got 53.1% ✅
+- Day 5: Expected 38.1%, Got 38.1% ✅
+
+---
+
+## 📊 Recovery Timeline (Legs Day A - Nov 5, 2025)
+
+### Initial Post-Workout Fatigue
+| Muscle       | Volume (lbs) | Baseline (lbs) | Fatigue | Status |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)